src/PlatformSwitch/hooks.ts

```diff
@@ -0,0 +1,57 @@
+import { redirect } from '@graasp/sdk';
+
+/** Enumeration of available platforms */
+export enum Platform {
+  Builder = 'Builder',
+  Player = 'Player',
+  Library = 'Library',
+  Analytics = 'Analytics',
+}
```

Comment:
We have `Context` but I guess context is not a clear wording. Should we match one with the other?

Reply:
True, I though they were defined in each platform codebase initially. I think renaming `Context` to `Platform` is more precise. I think it would break a bunch of stuff though.

Reply:
Update: i've tried using `Context` but all things break since I'm depending on the value of the enum. I could be rewriting the list without the deprecations, but it would be the same as using `Platform`. I'll stick with this and open an issue in  `graasp-sdk` to unify in the future.